doc/development.rst

```diff
@@ -405,6 +405,15 @@ To use it in conjunction with a `Resource` and a file:
    mf.sync_to_resource()
    path = mf.get_remote_path()
 
+Unless constructed with `ManagedFile(..., detect_nfs=False)`, ManagedFile
+employs following heuristic to check if a file is on NFS and if so, foregoes
+the transfer and `get_remote_path()` just returns the local path:
```

Comment:
perhaps add "(which is identical to the remote path in that case)"

Reply:
done.